openfisca_core/periods.py

```diff
@@ -358,6 +358,41 @@ def __str__(self):
             if fragment is not None
             )
 
+    def __cmp__(self, other):
```

Comment:
An order on periods is used here : https://github.com/openfisca/openfisca-core/blob/master/openfisca_core/scenarios.py#L55 https://github.com/openfisca/openfisca-core/blob/master/openfisca_core/scenarios.py#L162

My opinion is that only instants should have an order, not periods. I may move the comparison operators from periods.py to scenarios.py and forbid implicit period ordering. Sounds cleaner to you ?